common/common-ui-internal/src/main/java/com/duckduckgo/common/ui/internal/experiments/visual/VisualDesignExperimentViewModel.kt

```diff
@@ -56,22 +54,11 @@ class VisualDesignExperimentViewModel @Inject constructor(
 
     init {
         viewModelScope.launch {
-            visualDesignExperimentDataStore.experimentState.collect { state ->
+            visualDesignExperimentDataStore.isExperimentEnabled.collect { isExperimentEnabled ->
                 _viewState.update {
                     it.copy(
-                        isBrowserThemingFeatureAvailable = state.isAvailable,
-                        isBrowserThemingFeatureEnabled = state.isEnabled,
-                    )
-                }
-            }
-        }
-
-        viewModelScope.launch {
-            visualDesignExperimentDataStore.navigationBarState.collect { state ->
-                _viewState.update {
-                    it.copy(
-                        isNavigationBarAvailable = state.isAvailable,
-                        isNavigationBarEnabled = state.isEnabled,
+                        isBrowserThemingFeatureAvailable = true,
```

Comment:
In here's we're defining that the toggle in the "Experimental UI Settings" is visible. If in the future we have more toggles that depend on the overarching one, we can make them visible/invisible here depending on whether the parent is enabled. cc @joshliebe